app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,88 @@
+class SubscriptionsController < ApplicationController
+  before_action :set_subscription, only: %i[ show edit update destroy ]
+  before_action :check_logon
+  before_action :set_forum, only: %w[new create]
+  # GET /subscriptions or /subscriptions.json
+  def index
+    @forums = Forum.joins(:subscriptions).where(subscriptions: {user_id: @user.id}).order(:priority)
```

Comment:
Navigating to the Subscriptions page results in an error:

![Screenshot 2024-08-28 at 3 11 42 AM](https://github.com/user-attachments/assets/dce83d24-46b0-496f-a433-78456c1f9951)

Hint: Where is `@user` defined in this controller? Is there another instance variable that you could use? This is a showstopper for this lesson!